lib/logstash/filters/yaml.rb

```diff
@@ -0,0 +1,99 @@
+# encoding: utf-8
+require "logstash/filters/base"
+require "logstash/namespace"
+require "logstash/timestamp"
+
+# This is a YAML parsing filter. It takes an existing field which contains YAML and
+# expands it into an actual data structure within the Logstash event.
+#
+# By default it will place the parsed YAML in the root (top level) of the Logstash event, but this
+# filter can be configured to place the YAML into any arbitrary event field, using the
+# `target` configuration.
+class LogStash::Filters::Yaml < LogStash::Filters::Base
+
+  config_name "yaml"
+  milestone 1
```

Comment:
`milestone` is deprecated. We use version instead. Since this is a new plugin, you'll start with version `0.1.0` in the gemspec